### PR TITLE
Simplify use of search::diskindex::Fusion class.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@baldersheim : FYI

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

File: searchlib/src/vespa/searchlib/diskindex/field_merger.cpp [hunk 1]
```diff
@@ -174,18 +177,20 @@ FieldMerger::renumber_word_ids_start()
         return false;
     }
     _word_aggregator = std::make_unique<WordAggregator>();
+    _word_heap->setup(4);
```
[the remark this answers is not in the document]
Comment (toregge) [reply searchlib/src/vespa/searchlib/diskindex/field_merger.cpp]: Fixed.

File: searchlib/src/vespa/searchlib/diskindex/field_merger.cpp [hunk 2]
```diff
@@ -356,6 +361,8 @@ FieldMerger::setup_merge_heap()
             _heap->initialAdd(reader.get());
         }
     }
+    _heap->setup(4);
```
[the remark this answers is not in the document]
Comment (toregge) [reply searchlib/src/vespa/searchlib/diskindex/field_merger.cpp]: Fixed.

**Review by geirst — APPROVED** [review]
👍

**Comment by baldersheim** [comment]
Test failure is unrelated

**Pull request merged**